### PR TITLE
[Data] [Scraper] Moital registries (manpower and service provider registries)

### PR DESCRIPTION
**Pull request by simonbor**

### CODE REVIEW

File: budgetkey_data_pipelines/pipelines/entities/moital/scraper.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import time
+import requests
+import logging
+from pyquery import PyQuery as pq
+from datapackage_pipelines.wrapper import ingest, spew
+
+from selenium import webdriver
+# from selenium.common.exceptions import TimeoutException
```
Comment (akariv) [inline budgetkey_data_pipelines/pipelines/entities/moital/scraper.py]: remove commented out code

File: budgetkey_data_pipelines/pipelines/entities/moital/pipeline-spec.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+moital_service_providers:
+  pipeline:
+    - run: add_metadata
+      parameters:
+        name: moital_service_providers
```
Comment (akariv) [inline budgetkey_data_pipelines/pipelines/entities/moital/pipeline-spec.yaml]: So, we have two datasets:
Service contractors: http://apps.moital.gov.il/WebServicesHandlers/ServiceContractors.aspx
Manpower contractors: http://apps.moital.gov.il/WebServicesHandlers/ManpowerContractors.aspx

we should keep names consistent - this dataset is called `moital_service_providers` but in fact scrapes `ManpowerContractors.aspx`...

File: budgetkey_data_pipelines/pipelines/entities/moital/pipeline-spec.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        title: Moital Service Providers
+    - run: scraper
```
Comment (akariv) [inline budgetkey_data_pipelines/pipelines/entities/moital/pipeline-spec.yaml]: Let's add the service contractors as well, and concatenate the two into one big list.

Comment (simonbor) [inline budgetkey_data_pipelines/pipelines/entities/moital/pipeline-spec.yaml]: Did you saw that the ServiceContractors page have one extra column?
In other words the list will contain empty values in the ManpowerContractors output.

**Comment by simonbor** [comment]
Can I push to master?

**Comment by akariv** [comment]
Simon, I can't see any fixes to the comments - did you push your code with the cr fixes?

**Comment by simonbor** [comment]
Hi,
Actually I can't remember if I push the changes to the master after I
committed them.
So, what can I check for now or what the commands I need to run?

>git push origin master // ?

If yes what can I do after?
Sorry for my may be stupid questions but it is my first git push to the
project...

Thanks,
Simon

**Pull request merged**